docs/examples/kwok-manage-nodes-and-pods.md

```diff
@@ -0,0 +1,107 @@
+# Manage nodes and pods with kwok
+
+## Kwok with args `--manage-all-nodes=true`
+
+Kwok will keep all nodes in the cluster ready
+
+## Kwok with args `--manage-nodes-with-annotation-selector=kwok.x-k8s.io/node=fake`
+
+Kwok will keep all nodes with annotaion `kwok.x-k8s.io/node=fake` in the cluster ready
+
+## Create a node
+
+Node's status can be any value you want
+
+``` bash
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Node
+metadata:
+  annotations:
+    node.alpha.kubernetes.io/ttl: "0"
+    kwok.x-k8s.io/node: fake
+  labels:
+    beta.kubernetes.io/arch: amd64
+    beta.kubernetes.io/os: linux
+    kubernetes.io/arch: amd64
+    kubernetes.io/hostname: kwok-node-0
+    kubernetes.io/os: linux
+    kubernetes.io/role: agent
+    node-role.kubernetes.io/agent: ""
+    type: kwok
+  name: kwok-node-0
+spec:
+  taints: # Avoid scheduling actual running pods to fake Node
+    - effect: NoSchedule
+      key: kwok.x-k8s.io/node
+      value: fake
+status:
+  allocatable:
+    cpu: 32
+    memory: 256Gi
+    pods: 110
+  capacity:
+    cpu: 32
+    memory: 256Gi
+    pods: 110
+  nodeInfo:
+    architecture: amd64
+    bootID: ""
+    containerRuntimeVersion: ""
+    kernelVersion: ""
+    kubeProxyVersion: fake
+    kubeletVersion: fake
+    machineID: ""
+    operatingSystem: linux
+    osImage: ""
+    systemUUID: ""
+  phase: Running
+```
+
+## Create an pods
+
```

Comment:
Now we create some Pods to verify if they can land on the previously created Nodes: